app/presenters/dashboard/notifications_presenter.rb

```diff
@@ -2,7 +2,7 @@ class Dashboard::NotificationsPresenter < Dashboard::MessagesPresenter
   IGNORED_NOTIFICATIONS = %w(csv_data_export daily_report weekly_report).freeze
 
   def initialize(notifications)
-    super notifications.where.not(system_name: IGNORED_NOTIFICATIONS, title: nil).order(created_at: :desc)
+    super notifications.where.not(system_name: IGNORED_NOTIFICATIONS).where.not(title: nil).order(created_at: :desc)
```

Comment:
doesn't `not` revert the condition?

Reply:
I think it's right, check https://www.bigbinary.com/blog/rails-6-deprecates-where-not-working-as-nor-and-will-change-to-nand-in-rails-6-1

Reply:
You are right, I missed the first `not` :(